### PR TITLE
Guide, list and templates

### PR DESCRIPTION
**Pull request by Desser**

### CODE REVIEW

File: templates/HowToStart.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+2. Choose project to add from [project_list.md](../templates/project_list.md) 
+3. Create [Issue](https://github.com/keycryptovc/base/issues) with name of your project. Create branch in your forked repository 
+4. Gather and analyze info about project, fill [project template](project_template.md), save as "yourproject.md" in created branch
+5. Add your project to projects folder
+6. Add link to your project in [projects.md](../projects/projects.md)
+7. Create file(s) with key persons from your project, using [people template](people_template.md), save as "firstname_lastname.md"
+8. Add your person(s) files to people folder
+9. Add key persons to [people.md](../people/people.md)
+10. Link persons with their projects
+11. Pull request your branch
```
Comment (akhavr) [inline templates/HowToStart.md]: Create pull request for your branch

File: templates/project_list.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
+## Blockchain currencies and assets 
+1.  *[Bitcoin](/projects/bitcoin.md)*des
```
Comment (akhavr) [inline templates/project_list.md]: Perhaps, remove the projects from the list, that are already in the base?

Comment (Desser) [inline templates/project_list.md]: +

File: templates/HowToStart.md (hunk 1 above)
Comment (Desser) [inline templates/HowToStart.md]: +

File: templates/project_list.md (hunk 2 above)
Comment (Desser) [inline templates/project_list.md]: it appears so, that before taking project from project_list you need to check issues, to be sure it's not have been taken yet

**Comment by akhavr** [comment]
Closes #46

**Pull request merged**